internal/marketplaces/healthcheck.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2024 Stacklok, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package marketplaces
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/stacklok/minder/internal/marketplaces/bundles"
+	"github.com/stacklok/minder/internal/marketplaces/subscriptions"
+)
+
+// rather than model a `Marketplace` type, let's just build a throwaway
+// Healthcheck interface which loads the healthcheck bundle and calls the
+// correct methods on the `Bundle` and `SubscriptionService` interfaces
+type HealthcheckService interface {
```

Comment:
I'm not keen on having a HealthCheck specific interface nor code path here. It should be just another bundle.